src/layouts/navigation.ts

```diff
@@ -307,20 +306,48 @@ export const getNavigation = (section) => {
   nav.push({
     header: "Resources",
     children: [
-      { title: "Mainnet", href: "/resources/mainnet" },
-      { title: "Testnet", href: "/resources/testnet" },
-      { title: "RPC Endpoints", href: "/resources/" },
       {
-        title: "Community Pool Proposals",
-        href: "/resources/governance/community-pool-proposals",
+        title: "Addresses",
```

Comment:
Is this okay as "Addresses" ? Maybe we can change it to "Contract Addresses"

Reply:
Sounds good to me.